simples3.go

```diff
@@ -168,8 +168,12 @@ func (s3 *S3) getURL(bucket string, args ...string) (uri string) {
 
 // SetEndpoint can be used to the set a custom endpoint for
 // using an alternate instance compatible with the s3 API.
+// If no protocol is included in the URI, defaults to HTTPS.
 func (s3 *S3) SetEndpoint(uri string) *S3 {
 	if len(uri) > 0 {
+		if ! strings.HasPrefix(uri, "http") {
```

Comment:
Hey, can you please run `go fmt` on this line

Reply:
Opps, sorry for that!